### ISSUE TITLE
No support for classes using Java14's multiline string in IntelliJ IDEA.

### ISSUE THREAD

**Issue by opporancisis** [google/google-java-format#484]
I have a project where I use new features of Java14 in some classes. I noticed that GJF doesn't work for those classes.
For example, GJF doesn't work for this class:
```java
public class     Test {
  String     test    = """
          abc
          """;
}
```
Though it works in this case:
```java
public class Test {
  String test = "abc";
}
```

IDEA version: Ultimate 2020.1.1
GJF: 1.7.0.5

**Comment by cushon** [comment]
cc @plumpy 

The latest release (1.8) supports Java 14 features when running on JDK 14, but there isn't a corresponding release of the IntelliJ plugin yet:

```
$ java -fullversion
openjdk full version "14.0.1+7"
$ java -jar google-java-format-1.8-all-deps.jar Test.java
public class Test {
  String test = """
          abc
          """;
}
```

**Comment by plumpy** [comment]
I just published a 1.8.0.0 version of the plugin which should fix this. Sorry for the delay!
